tests/test_init.py

```diff
@@ -1,16 +1,22 @@
+import sys
+
+from gameyamlspiderandgenerator.plugin.steam import Search as Steam
 from gameyamlspiderandgenerator.util.setting import config, set_config
 
-config({"proxy": {"http": 'http://127.0.0.1:7890', "https": 'socks5://127.0.0.1:7891'},
-        "api": {'google-play': 'a714b00383f0662a61b2e382d55c685f17015617aa7048972da58a756fb75e90',
-                'apple': 'a714b00383f0662a61b2e382d55c685f17015617aa7048972da58a756fb75e90'}
-        })
-set_config('hook', ["search"])
-set_config('plugin', ["steam", "itchio"])
+config(
+    {
+        "proxy": {"http": "http://127.0.0.1:7890", "https": "socks5://127.0.0.1:7891"},
+        "api": {
+            "google-play": "a714b00383f0662a61b2e382d55c685f17015617aa7048972da58a756fb75e90",
+            "apple": "a714b00383f0662a61b2e382d55c685f17015617aa7048972da58a756fb75e90",
+        },
+    }
```

Comment:
```suggestion
    {
        "proxy": {"http": ..., "https": ...},
        "api": {
            "google-play": ...,
            "apple": ...,
        },
    }
```

1. Omit config for `proxy` and `api`, move to somewhere else, or read from a config file